app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
```

Comment:
Hm, alright. Since this is URLHelper, then I guess it's ok. Somehow feels strange to me that a later part can override a (mandatory?) prefix.

If you keep the behaviour of `URI.join`, perhaps add a test case to document that?